kt/plugins/godot-gradle-plugin/src/main/kotlin/godot/gradle/projectExt/setupConfigurationsAndCompilations.kt

```diff
@@ -33,6 +33,7 @@ fun Project.setupConfigurationsAndCompilations() {
         with(it.dependencies) {
             add(dependencies.create("org.jetbrains.kotlin:kotlin-stdlib:${kotlinJvmExtension.coreLibrariesVersion}"))
             add(dependencies.create("com.utopia-rise:$godotLibraryArtifactName:${GodotBuildProperties.assembledGodotKotlinJvmVersion}"))
+            add(dependencies.create("com.utopia-rise:tools-common:${GodotBuildProperties.assembledGodotKotlinJvmVersion}"))
```

Comment:
Why do we add this one ?

Reply:
I need tools-common in the bootstrap, it has utilities required at runtime.
It also fixes an inconsistency. The boostrap generated for the editor had it, but not the one generated by the gradle build.

Reply:
We need to fix this after next release as we add kotlin poet to user classpath with this one.